onnxruntime/core/providers/cuda/rnn/cudnn_rnn_base.h

```diff
@@ -99,13 +99,14 @@ class CudnnRnnBase : public CudaKernel {
     w_data_cache_ = nullptr;
 
     size_t state_size;
+    auto default_cudnn_handle = DefaultCudnnHandle();
```

Comment:
GetCudnnHandle(context) ?

Reply:
this during kernel initialization phase, context is not initialized yet